lib/temporal.rb

```diff
@@ -81,6 +83,40 @@ def signal_workflow(workflow, signal, workflow_id, run_id, input = nil)
       )
     end
 
+    def await_workflow_result(workflow:, workflow_id:, run_id:, **args)
```

Comment:
Can `run_id` be omitted or specified as `nil` to indicate the latest run of the workflow should be used? If so, it's worth calling out that behavior in a comment.

Reply:
I'll add a test and find out!